packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.3.1
 
+* Added explanation on how to generate the pigeon communication layer and mockito mock objects.
```

Comment:
Nit: added -> adds (present tense is now recommended for consistency)

Reply:
You are right, I applied the necessary fix. Thank you.